lib/src/dotenv.dart

```diff
@@ -4,60 +4,76 @@ import 'dart:io';
 
 import 'package:meta/meta.dart';
 
-import 'parser.dart';
+part 'parser.dart';
+
+final _env = <String, String>{
+  ...Platform.environment,
+};
+
+@visibleForTesting
+Map<String, String> get loadedEnv => _env;
```

Comment:
Making the loaded environment `Map` fully available (even if just read only) would help to not have to update any existing types in apps.
Since `Platform.environment` is a plain old `Map<String, String>` It would be preferred if we can get access to the raw map